lib/wavefront-metrics-formatter.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const delta = require('./delta');
+const Histogram = require('metrics').Histogram;
+
+function counterPoint(counter, prefix, ts, tags) {
+  if (delta.hasDeltaPrefix(counter.name)) {
+    const value = counter.count;
+    const name = delta.resetAndGetName(counter, prefix, counter.name, value);
+    return pointLine('', name, '', value, ts, tags);
+  } else {
+    return pointLine(prefix, counter.name, '', counter.count, ts, tags);
+  }
+}
+
+function meterPoints(meter, prefix, ts, tags) {
```

Comment:
Optional change, but I was thinking about something like this:
```javascript
function meterPoints(meter, prefix, ts, tags) {
  return [
    [    '.count', meter.count],
    ['.mean_rate', meter.meanRate()],
    [  '.m1_rate', meter.oneMinuteRate()],
    [  '.m5_rate', meter.fiveMinuteRate()],
    [ '.m15_rate', meter.fifteenMinuteRate()]
  ].map(([suffix, value]) => pointLine(prefix, meter.name, suffix, value, ts, tags));
}
```
With slight changes, could be used to unify the meterPoints and timerPoints functions.

Reply:
👍